web/src/settings_data.ts

```diff
@@ -114,6 +114,10 @@ export function user_has_permission_for_group_setting(
     setting_name: string,
     setting_type: "realm" | "stream" | "group",
 ): boolean {
+    if (page_params.is_spectator) {
+        return true;
```

Comment:
I'm pretty sure this is supposed to be false  :)

Reply:
yeah, got squashed into the wrong commit ^^', fixed it up!